common.go

```diff
@@ -94,6 +100,41 @@ func UUIDv4() string {
 	return token.String()
 }
 
+// GenerateSecureToken generates a cryptographically secure random token encoded in base64.
+// It uses crypto/rand for randomness and base64.RawURLEncoding for URL-safe output.
+// If length is less than or equal to 0, it defaults to 32 bytes (256 bits of entropy).
+// Returns an error if the random source fails.
+func GenerateSecureToken(length int) (string, error) {
+	if length <= 0 {
+		length = 32
+	}
+	bytes := make([]byte, length)
+	if _, err := randRead(bytes); err != nil {
+		return "", fmt.Errorf("utils: failed to read random bytes for token: %w", err)
+	}
+	return base64.RawURLEncoding.EncodeToString(bytes), nil
+}
+
+// GenerateSecureTokenMust is a convenience wrapper that panics on failure.
+func GenerateSecureTokenMust(length int) string {
+	s, err := GenerateSecureToken(length)
+	if err != nil {
+		panic(err)
+	}
+	return s
+}
+
+// SecureToken generates a secure token with default 32 bytes of entropy.
+//
+// NOTE: SecureToken is a convenience wrapper that will panic if the
+// underlying secure random source (crypto/rand) fails. For callers that
+// prefer to handle failures explicitly, use GenerateSecureToken(length)
+// which returns (string, error), or use GenerateSecureTokenMust(length) to
+// get the panicking behavior directly.
+func SecureToken() string {
```

Comment:
I rather see this return (string, err) and not panic(). That way its easier to use all across Fiber.